apps/index.md

```diff
@@ -5,24 +5,22 @@ title: Overview
 header: REST API documentation
 ---
 
-## Introduction {#introduction}
+## ePages Now API Documentation {#introduction}
 
-The ePages API is based on RESTful principles, allowing you to code in whatever language you prefer.
+The ePages Now API is based on RESTful principles, allowing you to code in whatever language you prefer.
 It supports JSON media types and uses UTF-8 character encoding.
 The API uses the standard HTTP request types `GET`, `POST`, `PUT`, `PATCH` and `DELETE`.
 
-Making use of this API, you can create awesome apps or single shop connections.
-It is designed to give developers a convenient way to access data contained within ePages shops.
+Making use of this API, you can create awesome apps or single shop connections for our product [ePages Now](https://www.epages.com/us/sell-online/now/). ePages Base developers: Don't worry, the endpoints for our product ePages Base are also still covered.
+It is designed to give developers a convenient way to access data contained within ePages Now shops.
```

Comment:
With the insertion concerning ePages Base it is not quite clear what "it" actually is. Maybe start with "The API is designed to give ..."